### PR TITLE
Cleaning up the riscof makefile and fixing cwd issue

### PR DESCRIPTION
**Pull request by silabs-kjetil**
The riscof tool requires you to run the riscof tool from a certain path as the config file paths are relative to the location where you execute riscof from. So in this change we are more explicit on setting the current working directory before running any of the riscof commands.

Also removing much of the headers/banner from the makefile to make it more readable and compact.

### CODE REVIEW

**Comment by silabs-kjetil** [comment]
@silabs-hfegran, @silabs-robin and @MikeOpenHWGroup  can you have a look at this PR?

**Review by silabs-robin — APPROVED** [review]
Approving based on:
`cwd` issue seems like a real problem, deserving of a fix.
The cleanup seems to make it easier to quickly understand what is where in the code.

However, I will not merge until @MikeOpenHWGroup has had a look.
Maybe the other users of riscof want to take a peek?

**Comment by silabs-kjetil** [comment]
@silabs-hfegran and @silabs-robin this PR seems to be approved, is there anything missing before this PR can be merged?

**Pull request merged**
